### DISCUSSION TITLE
setRenderedNoteScale

### DISCUSSION THREAD

**Discussion by Refenger** [Leather128/LeathersFunkinEngine#305] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/121453687/218321302-6c7a7a18-1625-47b6-8e26-c7e070f3f78f.png)
different functions have the same name. the game crashes when im trying to use it

**Comment by Leather128** [comment, marked as answer]
yup because old kade code was great, have fun using those functions at all :D (btw even seperating them doesn't seem to fix it very much in newer versions sooooooooooooooo)

**Reply by Refenger** [reply]
can i somehow set the note scale using setProperty()?
